flink-runtime/src/test/java/org/apache/flink/runtime/testutils/MiniClusterResource.java

```diff
@@ -167,6 +175,11 @@ private void startMiniCluster() throws Exception {
                 new Configuration(miniClusterResourceConfiguration.getConfiguration());
         configuration.setString(
                 CoreOptions.TMP_DIRS, temporaryFolder.newFolder().getAbsolutePath());
+        configuration.setString(
+                ConfigConstants.METRICS_REPORTER_PREFIX
+                        + "test_reporter."
```

Comment:
This breaks tests that have already been using a reporter named "test_reporter", like the JobManagerMetricsITCase.
I'll rename it to something that is likely unique.